### ISSUE TITLE
[Accessibility]A11y_VSCode_ViewExplorer_ScreenReader: VoiceOver reads all the list items along with content "File Explorer , listbox" for "File Explorer" section.

### ISSUE THREAD

**Issue by AccessibilityTestingTeam-TCS** [Microsoft/vscode#52402]
**Environment Details:**
VSCode Version : 1.24.0
OS Version : HighSierra10.13.5

**Additional Details:**
MAS Violated : MAS 1.3.1
ScreenReader : VoiceOver

**Repro Steps:**
1. Launch VS Code while VoiceOver is ON.
2. Navigate to Activity Bar and select "Explorer" button.
3. Navigate to "File Explorer" section list box items using "Tab".
4. Read all the list box items.

**Actual:**
VoiceOver reads all the list box items along with content "File Explorer ,listbox". Unnecessary content is read out with each of the list box item. 

**Expected:**
VoiceOver should not read unnecessary content "File Explorer , listbox" with each item. It should read "listitems" not "listbox".

**Recommendations:**
Refer below link which is repository of bug fixes code snippets:
https://microsoft.sharepoint.com/teams/msenable/mas/pages/browse-fixes.aspx

**User Impact:**
The screenreader users will get unnecessary content read out with each list item in list box and which would be annoying if repeated each time.

**MAS Reference:**
MAS 1.3.1 - https://microsoft.sharepoint.com/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4}

**Attachment for Reference:**
[A11y_VSCode_ViewExplorer_VoiceOver_FileExplorerListbox.pptx](https://github.com/Microsoft/vscode/files/2118203/A11y_VSCode_ViewExplorer_VoiceOver_FileExplorerListbox.pptx)

Does this issue occur when all extensions are disabled?: Yes

**Comment by isidorn** [comment]
Dup of https://github.com/Microsoft/vscode/issues/52003
